lib/src/rules/give_deprecation_message.dart

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2018, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/visitor.dart';
+import 'package:linter/src/analyzer.dart';
+
+const _desc = r'Give a deprecation message, via `Deprecated("message")`.';
```

Comment:
Please don't use markdown in the description. It won't be translated before being displayed, except in the generated documentation.

Reply:
Should I just remove the backticks? Add an `@`?

Reply:
Seems like a reasonable choice to me, yeah.

Reply:
Done.